pkg/observability/trace.go

```diff
@@ -75,11 +88,11 @@ func Start(serviceName string) error {
 	return nil
 }
 
-// canSkipInit returns a bool representing if OtelStartEnvVar is set to false. This function is a helper for Start.
+// canSkipInit returns a bool representing if OtelStartEnvVar is set empty or not. This function is a helper for Start.
 // Instrumentation should *ONLY* be started if this function returns false
 func canSkipInit() bool {
 	startEnv := os.Getenv(OtelStartEnvVar)
-	return strings.ToLower(startEnv) == "false"
+	return strings.ToLower(startEnv) == "disabled"
```

Comment:
Don't you need to check for empty and return true in that case?

Reply:
Its just checking "disabled", now the docs match.